### PR TITLE
Write notes for wider context sections

### PR DESCRIPTION
**Pull request by drvinceknight**

### CODE REVIEW

File: src/chapters/04/main.tex [hunk 1]
```diff
@@ -397,9 +397,16 @@ \section{Solving with R}\label{sec:solving-with-R}
 
 \section{Research}\label{sec:research}
 
-TBA
-
+There are a number of further areas related to the study of as well as the use
+of differential equations. Topics omitted here include the actual solution
+approaches which in this chapter are taken
+care of using open source software. Chapters 9, 14 and 16
+of~\parencite{stewart2009probability} provide a good introduction to some of
```
Comment (geraintpalmer) [inline src/chapters/04/main.tex:404]: are you sure you meant this book?

File: src/chapters/03/main.tex [hunk 2]
```diff
@@ -719,8 +719,15 @@ \section{Wider context}\label{sec:wider_context}
...
 
-The blood supply chain is one of many areas within healthcare where discrete
-event simulation is used. A general overview is given
-in~\parencite{brailsford2009analysis}.
-
-% TODO Add a Hybrid paper.
+One domain where simulation is popularly used is in modelling healthcare systems.
+A general overview is given in~\parencite{brailsford2009analysis} where uses
+include resource utilisation, human behaviour, and workforce management. Simulation
+modelling is often combined with other methodologies such as statistical modelling.
```
Comment (drvinceknight) [inline src/chapters/03/main.tex:725]:
```suggestion
include resource utilisation, human behaviour, and workforce management. 
```

Not really specific to simulation?

File: src/chapters/03/main.tex [hunk 3]
```diff
@@ -719,8 +719,15 @@ \section{Wider context}\label{sec:wider_context}
...
+
+In order to be able to fully capture all the relevant details of the system to be
```
Comment (drvinceknight) [inline src/chapters/03/main.tex:727]: Nice paragraph.

File: src/chapters/04/main.tex (hunk 1 above)
Comment (drvinceknight) [reply src/chapters/04/main.tex:404]: No the Calculus book is also by a Stewart.

File: src/chapters/09/main.tex [hunk 4]
```diff
@@ -745,5 +745,30 @@ \section{Solving with R}\label{sec:solving-with-R}
...
+include greedy searches, tabu searches, simulated annealing, genetic algorithms,
+and ant colony optimisation. They are usually employed when the problem is too
+large or complex to use exact methodologies.
+
+The Travelling Salesman Problem, described in this chapter, is a classic example
+of one of these problems, formally described first in~\ref{merger1932}, although
+thought to have been discussed informally centuries before.
+It is an example of a large number of types of problems collectively known as
+vehicle routing problems, which often require heuristic methods for their
+solutions. An survey is given in~\parencite{braekers2016vehicle}. Variations of
```
Comment (drvinceknight) [inline src/chapters/09/main.tex:763]: `A survey`

File: src/chapters/09/main.tex [hunk 5]
```diff
@@ -745,5 +745,30 @@ \section{Solving with R}\label{sec:solving-with-R}
...
+the problem include multiple, heterogeneous and/or capacitated vehicles, and
+stochastic or time-dependent travel times. A recently adaptation of the problem
```
Comment (drvinceknight) [inline src/chapters/09/main.tex:765]: `A recent`

File: src/chapters/09/main.tex [hunk 6]
```diff
@@ -745,5 +745,30 @@ \section{Solving with R}\label{sec:solving-with-R}
...
+is the green vehicle routing problem~\parencite{moghdani2021green}, where the
+cost function includes considerations around green house gas emissions and other
```
Comment (drvinceknight) [inline src/chapters/09/main.tex:767]: `consideration of green house`

File: src/chapters/09/main.tex [hunk 7]
```diff
@@ -745,5 +745,30 @@ \section{Solving with R}\label{sec:solving-with-R}
...
+pollutants.
+
+For more diverse applications of heuristic methods,
+consider~\parencite{lewis2016creating} which describes a tabu search algorithm
+for finding seating plans for a wedding; and \parencite{tong2013modeling} where
+a genetic algorithm is used to build a prediction model predicting locations of
```
Comment (drvinceknight) [inline src/chapters/09/main.tex:773]: `prediction model for locations`

File: src/chapters/04/main.tex (hunk 1 above)
Comment (drvinceknight) [reply src/chapters/04/main.tex]: This is fixed.

**Pull request merged**